src/main/java/io/FileIO.java

```diff
@@ -20,9 +21,23 @@ public String workingDirectory() {
     return workingDirectory;
   }
 
+  public boolean isPathFile(String path) throws IOException {
+    File checkFile = new File(getPath(path));
+    return checkFile.isFile();
+  }
+
   public byte[] getContent(String fileName) throws IOException {
-      Path path = Paths.get(getPath(fileName));
-      return Files.readAllBytes(path);
+    Path path = Paths.get(getPath(fileName));
+    return Files.readAllBytes(path);
+  }
+
+  public ArrayList<String> getFiles(String directory) throws IOException {
+    File dir = new File(getPath(directory));
+    if (!dir.isDirectory()) {
+      throw new IOException();
+    }
+    ArrayList<String> list =  new ArrayList<String>(Arrays.asList(dir.list()));
```

Comment:
`dir.list()` should return a `Collection`. That's a little more generic than `ArrayList` - which you can probably pass around if the only thing you're ever doing to it is iterating. It will also get rid of the cast/wrap.